MalwareAnalyzer/views/domain_check.py

```diff
@@ -146,13 +146,11 @@ def get_domains(urls):
     try:
         domains = []
         for url in urls:
-            parsed_uri = urlparse(url)
-            if parsed_uri.port:
-                url = parsed_uri.hostname
-            if parsed_uri.port or not parsed_uri.scheme:
-                url = '//' + url
-                parsed_uri = urlparse(url)
-            domain = '{uri.netloc}'.format(uri=parsed_uri)
+            parse_uri = urlparse(url)
+            if (parse_uri.port or not parse_uri.scheme) and parse_uri.hostname:
```

Comment:
why explicit check for `parse_uri.hostname`?  At this level it already after URL extraction, it must have a hostname?

Reply:
I see sometime doamin with empty value which will make the function crash, don't know why it is possible, so I prefer add this test to secure the function

Reply:
in google one apk you have one malformed  url which reports empty scheme, empty netloc and path=www.google.com/visualsearch/lh/" and that's make the function crash if hostname is not checked

Reply:
okey. sounds good.